modules/apps/portal-workflow/portal-workflow-test/src/testFunctional/tests/WorkflowMetricsAllItemsList.testcase

```diff
@@ -0,0 +1,455 @@
+
+	}
+
+	@description = "LPS-92053 - Verify that, in the item detail modal, when an item is completed, the summary section should hide the info Current Step and replace it with End Date"
+	@ignore = "Test Stub"
+	@priority = "3"
+	test SummarySectionDisplaysEndDateInsteadOfCurrentStep {
+
+		// TODO LRQA-67763 SummarySectionDisplaysEndDateInsteadOfCurrentStep pending implementation
+
+	}
+
+	@description = "LPS-86652 - Verify that hovering the cursor on the process step, when it contains more than 1 option, the toolip will display a bullet list"
+	@ignore = "Test Stub"
+	@priority = "3"
+	test TooltipDisplaysBulletListWhenMultipleItems {
+
+		// TODO LRQA-67732 TooltipDisplaysBulletListWhenMultipleItems pending implementation
+
+	}
+
+	@description = "LPS-92053 - Verify that the Go to submission page button, on hover, displays a tooltip saying Open page in new tab"
+	@ignore = "Test Stub"
+	@priority = "3"
+	test TooltipIsDisplayedWhenHoveringButton {
+
+		// TODO LRQA-67741 TooltipIsDisplayedWhenHoveringButton pending implementation
+
+	}
+
+	@description = "LPS-108138 - Verify that, in All Items page, when the users select an instance that can't be transitioned by them, the option to transition is not available to be selected"
+	@ignore = "Test Stub"
+	@priority = "4"
+	test TransitionOptionIsNotAvailableWhenUserCanNotTransitionInstance {
+
+		// TODO LRQA-67744 TransitionOptionIsNotAvailableWhenUserCanNotTransitionInstance pending implementation
+
+	}
+
+	@description = "LPS-98517 - Verify that the usernames in the result table are separated by a comma when there is more than one assignee per item"
+	@ignore = "Test Stub"
+	@priority = "3"
+	test UsernamesAreSeparatedByComma {
+
+		// TODO LRQA-67751 UsernamesAreSeparatedByComma pending implementation
+
+	}
+
+	@description = "LPS-98517 - Verify that, in the item detail modal, the usernames are separated by a comma when there is more than one assignee per item under the Process Details section"
+	@ignore = "Test Stub"
+	@priority = "3"
+	test UsernamesAreSeparatedByCommaInItemDetailModal {
+
+		// TODO LRQA-67746 UsernamesAreSeparatedByCommaInItemDetailModal pending implementation
+
+	}
+
+	@description = "LPS-92053 - Verify that, in the item detail modal, Pause instead of due date, the UI will inform SLA Paused sentence next to the SLA name"
+	@ignore = "Test Stub"
+	@priority = "4"
+	test WhenSlaIsPausedSlaPausedIsDisplayedInsteadOfDueDate {
+
+		// TODO LRQA-67770 WhenSlaIsPausedSlaPausedIsDisplayedInsteadOfDueDate pending implementation
+
+	}
+
+	@description = "LPS-92053 - Verify that, in the item detail modal, Open when an SLA is paused, the SLA status will be related to the moment when it triggered the pause timer"
+	@ignore = "Test Stub"
+	@priority = "4"
+	test WhenSlaIsPausedSlaStatusIsRelatedToThePauseTimer {
+
+		// TODO LRQA-67767 WhenSlaIsPausedSlaStatusIsRelatedToThePauseTimer pending implementation
+
+	}
+
```

Comment:
Hi @leticiachiu !
There is one stub missing. Could you, please, add it?
```
@description = "LPS-87571 - Verify that the filter query only be applied after user moves the cursor out of the dropdown cardr"	
@ignore = "Test Stub"	
@priority = "3"	test FilterIsAppliedAfterCursorMovesOut {		
// TODO LRQA-67773 FilterIsAppliedAfterCursorMovesOut pending implementation	}
```
You can find it in line 603 of the Test Map.

Reply:
hello @anabuchmann! you can find this stub on line 158 :)